torch/_inductor/compile_fx.py

```diff
@@ -576,13 +586,15 @@ def is_aligned(storage_offset, dtype):
         if isinstance(inputs[i], torch.Tensor)
         and (
             i not in static_input_idxs
-            or not is_aligned(inputs[i].storage_offset(), inputs[i].dtype)
+            or not is_aligned(inputs[i].storage_offset(), inputs[i].dtype)  # type: ignore[union-attr]
```

Comment:
Might be cleaner to enumerate over `inputs` so that we get type refinement on the local tensor. I think because we are type checking on `inputs[i]` instead it doesnt type refine and you need to add the ignores

Reply:
Yes, fixed. Thanks.